modules/Ecommerce/Banner/Requests/ExportBannerRequest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Modules\Ecommerce\Banner\Requests;
+
+use App\Http\Requests\BaseExportRequest;
+
+class ExportBannerRequest extends BaseExportRequest
+{
+    protected function getModelSpecificRules(): array
+    {
+        return [
+            'name' => 'sometimes|string|max:255',
+        ];
+    }
+
+    protected function getModelSpecificFilters(): array
+    {
+        $filters = [];
+        
+        if ($this->has('name')) {
```

Comment:
Consider using `filled()` instead of `has()` to ensure the 'name' parameter is not empty. This would prevent empty strings from being passed as filters.

```suggestion
        if ($this->filled('name')) {
```